database/connect.js

```diff
@@ -18,4 +20,5 @@ connection.connect(function(err){
   console.log('Ikea details Connected');
 });
 
+
```

Comment:
where i can see the config setting for dB connection

Reply:
It should be in that file. It is in the first 15 lines.

Reply:
sorry, i did'nt click on the button .

Reply:
Perfect